estimagic/tests/sensitivity/test_msm_sensitivity.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+import pandas as pd
+import pytest
+from numpy.testing import assert_array_almost_equal as aaae
+from scipy import stats
+
+from estimagic.config import TEST_DIR
+from estimagic.differentiation.derivatives import first_derivative
+from estimagic.inference.msm_covs import cov_optimal
+from estimagic.sensitivity.msm_sensitivity import calculate_actual_sensitivity_to_noise
+from estimagic.sensitivity.msm_sensitivity import (
+    calculate_actual_sensitivity_to_removal,
+)
+from estimagic.sensitivity.msm_sensitivity import (
+    calculate_fundamental_sensitivity_to_noise,
+)
+from estimagic.sensitivity.msm_sensitivity import (
+    calculate_fundamental_sensitivity_to_removal,
+)
+from estimagic.sensitivity.msm_sensitivity import calculate_sensitivity_to_bias
+from estimagic.sensitivity.msm_sensitivity import calculate_sensitivity_to_weighting
+
+
```

Comment:
Maybe we could add a test that checks some random edge cases. For example when all the variance is one moment only etc?

Reply:
Would postpone that for now.